app/views/coronavirus_form/business_details.html.erb

```diff
@@ -55,8 +57,9 @@
   heading_size: "s",
   name: "company_location",
   error_message: error_items('company_location'),
-  items: t('coronavirus_form.questions.business_details.company_location.options').map do |_, item|
+  items: t('coronavirus_form.questions.business_details.company_location.options').map.with_index do |(_, item), index|
     {
+      id: ("business_details.company_location" if index == 0),
```

Comment:
This might be nicer to do using the translation text rather than index, just in case these were reordered..

Reply:
Re-order of items can be done without issues, the aim is to get the first item, whichever that is.